src/checks.rs

```diff
@@ -29,17 +30,20 @@ impl fmt::Display for Warning {
     }
 }
 
+// This trait is used for checks which needs to know of only a single line
 trait Check {
     fn run(&mut self, line: LineEntry) -> Option<Warning>;
 }
 
+// Checklist for checks which needs to know of only a single line
 fn checklist() -> Vec<Box<dyn Check>> {
     vec![
         Box::new(incorrect_delimiter::IncorrectDelimiterChecker::default()),
         Box::new(leading_space::LeadingSpaceChecker::default()),
         Box::new(key_without_value::KeyWithoutValueChecker::default()),
         Box::new(lowercase_key::LowercaseKeyChecker::default()),
         Box::new(spaces_around_equal::SpacesAroundEqualChecker::default()),
+        Box::new(duplicated_keys::DuplicatedKeysChecker::default()),
```

Comment:
Please move it to the top of this list.

Reply:
I did, could you tell me what the logic here should be? It's not very clear and doesn't seem alphabetical?

Reply:
Just sorting in alphabetical order 😃